posts/inside-rust/2022-05-04-CTCFT-may.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: CTCFT 2022-05-16 Agenda
+author: Rust CTCFT Team
+---
+
+The next ["Cross Team Collaboration Fun Times" (CTCFT)][CTCFT] meeting will take
+place on Monday, 2022-05-16 at **11am US Eastern Time** ([click to see in your
+time zone][timezone]). You’ll find the full details (along with a calendar
+event, zoom details, etc) [on the CTCFT website][CTCFT-meeting].
+
+[CTCFT]: https://rust-lang.github.io/ctcft/
+[timezone]: https://everytimezone.com/s/6c2a0d08
+[CTCFT-meeting]: https://rust-lang.github.io/ctcft/meetings/2022-05-16.html
+
+## Agenda
+
+The theme for this month's CTCFT is **Embedded Rust**. We'll hear from some
+members of the Rust Embedded Working Group about the state of the ecosystem, as
```

Comment:
Do you mean since @Dirbaio isn't part of the governance structure? I could improve this if there is a better way to phrase it, but I feel like it still represents the context well enough :)